u8timeseries/backtesting/backtesting.py

```diff
@@ -259,3 +260,193 @@ def plot_residuals_analysis(residuals: TimeSeries, num_bins: int = 20):
     ax3.set_ylabel('ACF value')
     ax3.set_xlabel('lag')
     ax3.set_title('ACF')
+
+
+def backtest_gridsearch(model_class: type,
+                        parameters: dict,
+                        train_series: TimeSeries,
+                        val_series: Optional[TimeSeries] = None,
+                        fcast_horizon_n: int = 5,
```

Comment:
I think we should maybe force the user to select a forecast horizon (i.e. not specify a default value)